docs-ref-services/advisor.md

```diff
@@ -23,16 +23,30 @@ With Advisor, you can:
 - Improve the performance, security, and high availability of your resources, as you identify opportunities to reduce your overall Azure spend.
 - Get recommendations with proposed actions inline.
 
-## Install the modules with npm
+## Management
 
 Use npm to install the Azure Advisor modules for Node.js
 
-### Management
+### Install the modules with npm
 ```bash
 npm install azure-arm-advisor
 ```
-##Example
-
-##Samples
+## Example
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const advisorManagement = require('azure-arm-advisor');
+
+msRestAzure.interactiveLogin().then( credentials => {
+  const client = new advisorManagement(credentials, 'your-subscription-id');
+  client.recommendations.list().then(recommendations => {
+    console.log('List of recommendations:');
+    console.dir(recommendations, {depth: null, colors: true});
+  });
+}).catch(err => {

```

Comment:
For most demos I applaud error handling. But for the APIs I wonder if it is better and more readable to err on the side of brevity/readability and remove the error handling.

Thoughts?